examples/api/lib/cupertino/context_menu/cupertino_context_menu.1.dart

```diff
@@ -0,0 +1,115 @@
+
+class ContextMenuApp extends StatelessWidget {
+  const ContextMenuApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      theme: CupertinoThemeData(brightness: Brightness.light),
+      home: ContextMenuExample(),
+    );
+  }
+}
+
+class ContextMenuExample extends StatelessWidget {
+  const ContextMenuExample({super.key});
+
+  Animation<Decoration> _boxDecorationAnimation(Animation<double> animation) {
+    return DecorationTween(
+      begin: BoxDecoration(
+          color: CupertinoColors.systemYellow,
+          boxShadow: const <BoxShadow>[],
+          borderRadius: BorderRadius.circular(20.0),
+      ),
+      end: BoxDecoration(
+        color: CupertinoColors.systemYellow,
+        boxShadow: CupertinoContextMenu.kEndBoxShadow,
+        borderRadius: BorderRadius.circular(20.0),
+      ),
+    ).animate(
+      CurvedAnimation(
+        parent: animation,
+        curve: Interval(
+          0.0,
+          CupertinoContextMenu.animationOpensAt,
+      ))
+    );
+  }
+
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: const CupertinoNavigationBar(
+        middle: Text('CupertinoContextMenu Sample'),
+      ),
+      child: Center(
+        child: SizedBox(
+          width: 100,
+          height: 100,
+          child: CupertinoContextMenu.builder(
+            actions: <Widget>[
+              CupertinoContextMenuAction(
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+                isDefaultAction: true,
+                trailingIcon: CupertinoIcons.doc_on_clipboard_fill,
+                child: const Text('Copy'),
+              ),
+              CupertinoContextMenuAction(
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+                trailingIcon: CupertinoIcons.share,
+                child: const Text('Share'),
+              ),
+              CupertinoContextMenuAction(
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+                trailingIcon: CupertinoIcons.heart,
+                child: const Text('Favorite'),
+              ),
+              CupertinoContextMenuAction(
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+                isDestructiveAction: true,
+                trailingIcon: CupertinoIcons.delete,
+                child: const Text('Delete'),
+              ),
+            ],
+            builder:(BuildContext context, Animation<double> animation) {
+              final Animation<Decoration> boxDecorationAnimation =
+                _boxDecorationAnimation(animation);
+
+              return Container(
+                decoration:
+                  animation.value < CupertinoContextMenu.animationOpensAt ?
+                  boxDecorationAnimation.value : null,
```

Comment:
Nit: Usually the formatting I see in the framework would be more like this:

```dart
decoration: animation.value < CupertinoContextMenu.animationOpensAt
  ? boxDecorationAnimation.value
  : null,
```

Reply:
👍 I went looking around for an example and had trouble finding one.